packages/dashboard/tsconfig.json

```diff
@@ -0,0 +1,44 @@
+{
+    "compilerOptions": {
+        /* Visit https://aka.ms/tsconfig.json to read more about this file */
+        "target": "es2020",
+        "module": "es2020",
+        "jsx": "react",
+        "sourceMap": true,
+        "outDir": "$$ts-jest$$",
```

Comment:
why `$$ts-jest$$`? 
does it output only jest files?

Reply:
This is a remnant from initially setting up `tsconfig.json`, it can safely be removed. The `tsconfig` is only a guideline for our IDE (e.g. to resolve paths, enforce code styles), the actual compilation/transpilation is done by Babel (following our config).